server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -51,51 +51,42 @@ public class ResolvedPods {
         public List<String> getAllPodLibs(String platform) {
             Set<String> libs = new LinkedHashSet<>();
             for (PodSpec pod : pods) {
-                if (platform.contains("ios")) {
-                    libs.addAll(pod.libraries.ios);
-                }
-                else if (platform.contains("osx")) {
-                    libs.addAll(pod.libraries.osx);
-                }
+                libs.addAll(pod.libraries.get(platform));
             }
             return new ArrayList<String>(libs);
         }
 
         public List<String> getAllPodLinkFlags(String platform) {
             Set<String> flags = new LinkedHashSet<>();
             for (PodSpec pod : pods) {
-                if (platform.contains("ios")) {
-                    flags.addAll(pod.linkflags.ios);
-                }
-                else if (platform.contains("osx")) {
-                    flags.addAll(pod.linkflags.osx);
-                }
+                flags.addAll(pod.linkflags.get(platform));
             }
             return new ArrayList<String>(flags);
         }
 
+        public List<String> getAllPodResources(String platform) {
```

Comment:
Get all pod resources. Each resource has a path relative to the pod dir. Return absolute paths.